src/index.ts

```diff
@@ -62,7 +76,7 @@ export interface MongoCrypt {
        * When true, creates a `mongocrypt_ctx_explicit_encrypt_expression` context.
        * When false, creates a `mongocrypt_ctx_explicit_encrypt`
        */
-      expressionMode: boolean;
+      expressionMode?: boolean;
```

Comment:
@nbbeeken This came from one of your commits, I _think_ this can be a required parameter?

Reply:
Yeah it is not currently required because of the coercion Bailey points out and there's a bunch of tests that omit this and have TS red underlines as a result, but don't fail at runtime. But we can keep this as "required" for TS sake.

Reply:
What tests currently have red because of this?  This is an implementation detail in the driver - all CSFLE tests use `encrypt()` or `encryptExpression()`

Reply:
@nbbeeken I see you reverted the optionality - were you going to adjust the C++ too?

Reply:
No, we should do that in another PR, this should just contain SSL-related changes. I should've left this alone in the first place. Not sure where I saw the red underlines, but I recall that was the inspiration

Reply:
I opened https://github.com/mongodb-js/mongodb-client-encryption/pull/37 for this.